pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -861,6 +907,14 @@ def has_ipu(self) -> bool:
     def use_ipu(self) -> bool:
         return isinstance(self.accelerator, IPUAccelerator)
 
+    @property
+    def has_hpu(self) -> bool:
+        return isinstance(self.accelerator, HPUAccelerator) and isinstance(self.strategy, HPUStrategy)
+
+    @property
+    def use_hpu(self) -> bool:
+        return isinstance(self.accelerator, HPUAccelerator)
+
```

Comment:
We will be removing these properties so you can instead use their implementations directly.